src/views/List.jsx

```diff
@@ -61,16 +102,46 @@ export function List({ data, listPath }) {
 							X
 						</button>
 					</form>
-					<ul>
-						{filteredItems.map((item) => (
-							<ListItem
-								key={item.id}
-								id={item.id}
-								listPath={listPath}
-								itemData={item}
-							/>
-						))}
-					</ul>
+					{items.map((item) => (
+						<div key={item.id}>
+							<label>
+								<input
+									type="checkbox"
+									value={item.id}
+									onChange={() => {
+										// Handle checkbox change
+									}}
```

Comment:
Checking into the other issues branch the checkbox to buy is working, in this branch seems broken. could it be because of this onChange() do nothing?